app/src/main/java/org/mozilla/focus/settings/MozillaSettingsFragment.kt

```diff
@@ -88,10 +88,12 @@ class MozillaSettingsFragment :
     }
 
     override fun onSharedPreferenceChanged(sharedPreferences: SharedPreferences, key: String) {
-        TelemetryWrapper.settingsEvent(key, sharedPreferences.all[key].toString())
-        ProTips.tipsSettingChanged.record(
-            ProTips.TipsSettingChangedExtra(sharedPreferences.all[key] as Boolean)
-        )
+        if (isAdded && key != requireContext().getString(R.string.number_of_opened_tabs)) {
```

Comment:
Instead of `key != requireContext().getString(R.string.number_of_opened_tabs)` I think we should use `key == requireContext().getString(R.string.pref_key_homescreen_tips)` since is the only preference which can be changed from this screen and has as a value a boolean

Reply:
Done